README.md

```diff
@@ -52,11 +52,10 @@ file. This will take a few seconds in any case.
 
 ```shell
 $ lndinit -v init-wallet \
-    --secret-source=file \
```

Comment:
I'd like to keep the examples as explicit as possible, so please leave this in, even though it's the default.